### PR TITLE
cliGroundStation now returns a string

### PR DESCRIPTION
**Pull request by Taylorr82**

### CODE REVIEW

File: src/sat_cli.py [hunk 1]
```diff
@@ -34,7 +34,8 @@ def cli():
         try:
             cmd = input("$ ")
             server, port, toSend = gs.getInput(cmd, inVal="{}.cli.send_cmd({},{})".format(gs.satellite, len(cmd), cmd))
-            gs.transaction(server, port, toSend)
+            ret = gs.transaction(server, port, toSend)
+            print(ret)
```
Comment (rcunrau) [inline src/sat_cli.py:38]: Do you want to say that it's a response code, or interpret the error code?

**Review by rcunrau — APPROVED** [review]

File: src/sat_cli.py (hunk 1 above)
Comment (Taylorr82) [reply src/sat_cli.py:38]: What do you mean? If there is an error, the sat_cli creates its own error string to send back which would be printed here

**Review by jmmabanta — APPROVED** [review]
LGTM!

File: src/sat_cli.py (hunk 1 above)
Comment (rcunrau) [reply src/sat_cli.py:38]: Sorry, I misread "data[2:]" as a number.

**Pull request merged**